EPCPortalWeb/Models/DataHandlerModel.cs

```diff
@@ -10,7 +10,20 @@ public class DataHandlerModel
         public string Postcode { get; set; }
         public int HouseNumber { get; set; }
 
+        public string APIAddress { get; set; }
 
+        public List<string> FilterAddressesByHouseNumber(int houseNumber, List<string> listOfAPIAddresses)
```

Comment:
Suggest using IEnumerable<string> or some other suitable interface instead of List, in both the return object and argument.